client/blocks/comment-detail/index.jsx

```diff
@@ -26,16 +28,21 @@ export class CommentDetail extends Component {
 		authorUsername: PropTypes.string,
 		commentContent: PropTypes.string,
 		commentDate: PropTypes.string,
-		commentId: PropTypes.number,
+		commentId: PropTypes.number.isRequired,
```

Comment:
We may need to relax isRequired for `commentId` and `siteId` when we visit this route on a clean cache.